src/main/kotlin/brs/db/sql/SqlDb.kt

```diff
@@ -41,6 +41,7 @@ internal class SqlDb(private val dp: DependencyProvider) : Db {
         return if (con == null) {
             DSL.using(cp, dialect, settings)
         } else {
+            // Using transaction connection. Use static statements. TODO why?
```

Comment:
https://github.com/burst-apps-team/burstcoin/commit/aa5b04a66fe55a39eb05ad3cdf5759669b3ba111#diff-d5cd48c303de3c01c50833e2442f62abR231

Reply:
Yeah, I don't remember why I did this..